PHC-Android/PHC/src/main/java/phc/android/Checkout/CheckoutFormFragment.java

```diff
@@ -45,8 +46,13 @@
     /** Parent layout that holds all checkbox views. */
     private ViewGroup mLayout;
 
-    /** Array of service display names. */
-    private ArrayList<String> mServicesNotReceived;
+    /** Array of services applied but not received . */
+    private ArrayList<String> mServices;
+    /** Used to keep track IDs of checkboxes for services **/
+    private ArrayList<CheckBox> checkBoxArray;
+
+    /** Array of checked services still would like to receive . */
+    private ArrayList<String> mServicesChecked;
 
```

Comment:
can you arrange and cluster these variables in the order that they appear?
also could you remove the      /** currently not used because comment box is not yet set up  **/